### ISSUE TITLE
Caching and extra talk

### ISSUE THREAD

**Issue by TheUntraceable** [EpikCord/EpikCord.py#2]
We'll need to add some sort of caching so we don't end up spamming the API a bunch of times. We'll need to sort this all out via Cache Managers, I'll see what I can think of. So far I think I *could* make some sort of message command system via some ClientOption like `command_system: Union[Interaction, Message]` so that there's versatility between the 2.

**Comment by Jerrydotpy** [comment]
Did we not fixed this yet? We have a ratelimiter

**Comment by TheUntraceable** [comment]
No. Also I scrapped the message.content command system idea. Maybe, we'll see ;)

**Comment by nav-github01001** [comment]
We did this right

**Comment by TheUntraceable** [comment]
No.

**Comment by EmreTech** [comment]
What do you mean "no"? You clearly did do this. I saw the cache manager system in the source code.

**Comment by TheUntraceable** [comment]
There are still loads of cache managers I need to make, for roles, other stuff, I should've explained that
